packages/trader/src/config.js

```diff
@@ -89,7 +89,7 @@ const getSocketURL = () => {
 
         server_url = `${server}.binaryws.com`;
     }
-    return `wss://${server_url}/websockets/v3`;
+    return server_url;
```

Comment:
how come this is changed?

Reply:
We used to pass the endpoint to the DerivAPI constructor, however we changed this later to send a `WebSocket` instance to the constructor. This change makes it easier to switch between the two cases.